workers/unity/Packages/com.improbable.gdk.tools/DevAuthTokenUtils.cs

```diff
@@ -9,16 +9,26 @@ namespace Improbable.Gdk.Tools
 {
     public static class DevAuthTokenUtils
     {
+        public static string DevAuthToken => PlayerPrefs.GetString(DevAuthTokenKey);
+
+        private static string DevAuthTokenAssetPath =>
+            Path.Combine("Assets", GdkToolsConfiguration.GetOrCreateInstance().DevAuthTokenDir, "DevAuthToken.txt");
+
         private static readonly string JsonDataKey = "json_data";
+        private static readonly string ErrorKey = "error";
         private static readonly string TokenSecretKey = "token_secret";
+        private static readonly string DevAuthTokenKey = "devAuthTokenSecret";
```

Comment:
maybe prepend the key names to indicate whether they are used to get a value from json or put it into player preferences